mitodl/salt-ops
# feat: enable blog, webinars, course topic dropdown

# What are the relevant tickets?




Closes https://github.com/mitodl/hq/issues/3080

# Description (What does it do?)

Enables header links for xPro.

# Screenshots (if appropriate):

<img width="1344" alt="Screen Shot 2023-11-30 at 1 53 21 PM" src="https://github.com/mitodl/salt-ops/assets/52656433/5ccb71cc-70cc-45d6-af23-c6bed8794734">


# How can this be tested?

Enable these flags locally and verify that Header Links for Webinar, Blog, and Courses are visible.
